### PR TITLE
chore: `package.json` refactors

### PR DESCRIPTION
**Pull request by privatenumber**
Some minor refactors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,8 @@
   "name": "aicommits",
   "version": "1.0.2",
   "description": "Writes your git commit messages for you with AI",
-  "main": "bin/index.js",
-  "bin": {
-    "aicommits": "bin/index.js"
-  },
+  "files": ["bin"],
```
Comment (privatenumber) [inline package.json:5]: Using `files` to select publish files will be more future proof than block-listing each new file via `.npmignore`

File: package.json [hunk 2]
```diff
@@ -2,10 +2,8 @@
...
+  "bin": "bin/index.js",
```
Comment (privatenumber) [inline package.json:6]: If `bin` is a string, it will be the default binary.

File: tsconfig.json [hunk 3]
```diff
@@ -7,5 +7,4 @@
     "forceConsistentCasingInFileNames": true,
     "skipLibCheck": true /* Skip type checking all .d.ts files. */
   },
-  "exclude": ["node_modules"]
```
Comment (privatenumber) [inline tsconfig.json:10]: Ignored by default

**Comment by Nutlope** [comment]
Thanks for the contribution @privatenumber! Love how you commented on all your changed

**Pull request merged**